docs/tutorials/rerank/rerank_pipeline_with_Amazon_Rerank_model_on_Amazon_Bedrock.md

```diff
@@ -0,0 +1,622 @@
+# Topic
+
+[Reranking pipeline](https://opensearch.org/docs/latest/search-plugins/search-relevance/reranking-search-results/) is a feature released in OpenSearch 2.12.
+It can rerank search results, providing a relevance score for each document in the search results with respect to the search query.
```

Comment:
```suggestion
It can rerank search results, providing a relevance score with respect to the search query for each matching document.
```

Reply:
Thanks, I'm going to merge your recommendation on next commit.

Reply:
fixed on 9482b472b2bc68bba61267c87f484967362e2c5b